### PR TITLE
removed bower, added libman support.

### PR DESCRIPTION
**Pull request by bariscaglar**
Libman is the recommended way to manage client-side dependencies.

### CODE REVIEW

File: src/Microsoft.IIS.Administration/Microsoft.IIS.Administration.csproj [hunk 1]
```diff
@@ -54,6 +54,7 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="1.0.*" />
     <PackageReference Include="Microsoft.AspNetCore.StaticFiles" Version="1.0.*" />
     <PackageReference Include="Microsoft.AspNetCore.Diagnostics" Version="1.0.*" />
+    <PackageReference Include="Microsoft.Web.LibraryManager.Build" Version="1.0.*" />
```
Comment (shirhatti) [inline src/Microsoft.IIS.Administration/Microsoft.IIS.Administration.csproj]: @jimmyca15 Have we always had wildcards in dependency versions? It seems super scary not to have build determinism.

Comment (jimmyca15) [reply src/Microsoft.IIS.Administration/Microsoft.IIS.Administration.csproj]: No that was a recent change. What is the concern?

Comment (shirhatti) [reply src/Microsoft.IIS.Administration/Microsoft.IIS.Administration.csproj]: Non-determinism. You can end up with different versions among devs and the CI server.

**Comment by bariscaglar** [comment]
Can I get a review here?

File: src/Microsoft.IIS.Administration/Microsoft.IIS.Administration.csproj (hunk 1 above)
Comment (bariscaglar) [reply src/Microsoft.IIS.Administration/Microsoft.IIS.Administration.csproj]: Current version has all the specific versions, including the Microsoft.Web.LibraryManager.Build one.

**Comment by shirhatti** [comment]
Related to this PR, I'd like to see use Subresource Integrity especially if we're getting our static assets from an external CDN.

I'd add an integrity check here- https://github.com/Microsoft/IIS.Administration/blob/84391be8f874bce49433d34b8188ec4c570262a2/src/Microsoft.IIS.Administration/Views/Shared/_Layout.cshtml#L32

**Pull request merged**